File: Day 5/loops_v1.scala
object Solution {
    def main(args: Array[String]) {
        var numOfTests = scala.io.StdIn.readInt()
        for (i <- 1 to numOfTests) {
            var Array(a, b, n) = scala.io.StdIn.readLine.split(" ").map(_.toInt)
            var currentSum = a
            for (j <- 0 to n - 1) {
                currentSum += scala.math.pow(2, j).toInt * b
                System.out.print(currentSum.toString + " ")
            }
            System.out.println("")
        }
    }
}

File: Day 5/loops_v1.js
function processData(input) {
    var lines = input.split('\n');
    var numOfTests = parseInt(lines[0]);
    for (var i = 1; i <= numOfTests; ++i) {
        var nums = lines[i].split(' ')
        var a = parseInt(nums[0]), b = parseInt(nums[1]), N = parseInt(nums[2]);
        var currentSum = a;
        for (var j = 0; j < N; ++j) {
            currentSum += Math.pow(2, j) * b;
            process.stdout.write(currentSum.toString() + " ");
        }
        process.stdout.write("\n");
    }
} 

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});